manim/scene/scene_file_writer.py

```diff
@@ -712,6 +716,13 @@ def flush_cache_directory(self):
             {"par_dir": self.partial_movie_directory},
         )
 
+    def write_subcaption_file(self):
+        """Writes the subcaption file."""
+        subcaption_file = Path(config.output_file).with_suffix(".srt")
+        with open(subcaption_file, "w") as f:
+            f.write(srt.compose(self.subcaptions))
+        logger.info(f"Subcaption file has been written as {subcaption_file}")
```

Comment:
This function is supposed to stay untested, right?

Reply:
I think this is fine, yes. There is nothing exotic going on here, and the more critical part is the creation of the subcaptions.